src/pruna/algorithms/quantization/quanto.py

```diff
@@ -102,30 +105,32 @@ def _apply(self, model: Any, smash_config: SmashConfigPrefixWrapper) -> Any:
             The quantized model.
         """
         imported_modules = self.import_algorithm_packages()
-
         if hasattr(model, "unet"):
             working_model = model.unet
         elif hasattr(model, "transformer"):
             working_model = model.transformer
         else:
             working_model = model
 
-        weights = getattr(imported_modules["optimum"].quanto, smash_config["weight_bits"])
-        if smash_config["act_bits"] is not None:
-            activations = getattr(imported_modules["optimum"].quanto, smash_config["act_bits"])
-        else:
-            activations = None
+        imported_modules = self.import_algorithm_packages()
+
+        weights = getattr(imported_modules["quanto"], smash_config["weight_bits"])
+        activations = (
+            getattr(imported_modules["quanto"], smash_config["act_bits"])
+            if smash_config["act_bits"] is not None
+            else None
+        )
 
         try:
-            imported_modules["quantize"](working_model, weights=weights, activations=activations)
+            quantize(working_model, weights=weights, activations=activations)
```

Comment:
use from imported_modules
```suggestion
            imported_modules["quantize"](working_model, weights=weights, activations=activations)
```